includes/Includes_Neopets_Auctions/main.user.js

```diff
@@ -0,0 +1,95 @@
+// ==UserScript==
+// @name        Neopets : Includes : Auctions
+// @namespace   github.com
+// @include     nowhere
+// @exclude     *
+// @version     1
+// @grant       GM_xmlhttpRequest
+// @grant       GM_getResourceText
+// @require     http://code.jquery.com/jquery-latest.js
+// @require     https://github.com/knadh/localStorageDB/raw/master/localstoragedb.min.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_XPath/63808.user.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_HttpRequest/56489.user.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_Neopets_%5BBETA%5D/main.user.js?2387
+// ==/UserScript==
+
+// auctions.phtml?type=placebid
+// { _ref_ck : neo.ck, amount : 245000, auction_id : auctionid }
+
+var Auctions = function(page){
```

Comment:
Consider reading [Coding Style](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style).
I usually don't follow "one declaration per line", but all rules from this guide are very welcome.